Dockerfile-base

```diff
@@ -9,56 +9,81 @@ ENV baseDir .
 
 # Install dependencies
 # --------------------
-RUN apt-get update
-RUN apt-get install -y sudo
-RUN apt-get install -y build-essential wget net-tools unzip time imagemagick optipng strace nginx git python zlib1g-dev libpcre3-dev aspell aspell-en aspell-af aspell-am aspell-ar aspell-ar-large aspell-bg aspell-bn aspell-br aspell-ca aspell-cs aspell-cy aspell-da aspell-de aspell-el aspell-eo aspell-es aspell-et aspell-eu-es aspell-fa aspell-fo aspell-fr aspell-ga aspell-gl-minimos aspell-gu aspell-he aspell-hi aspell-hr aspell-hsb aspell-hu aspell-hy aspell-id aspell-is aspell-it aspell-kk aspell-kn aspell-ku aspell-lt aspell-lv aspell-ml aspell-mr aspell-nl aspell-nr aspell-ns  aspell-pa aspell-pl aspell-pt aspell-pt-br aspell-ro aspell-ru aspell-sk aspell-sl aspell-ss aspell-st aspell-sv aspell-tl aspell-tn aspell-ts aspell-uk aspell-uz aspell-xh aspell-zu
+RUN apt-get update \
+&&  apt-get install -y \
+      build-essential wget net-tools unzip time imagemagick optipng strace nginx git python zlib1g-dev libpcre3-dev \
+      aspell aspell-en aspell-af aspell-am aspell-ar aspell-ar-large aspell-bg aspell-bn aspell-br aspell-ca aspell-cs aspell-cy aspell-da aspell-de aspell-el aspell-eo aspell-es aspell-et aspell-eu-es aspell-fa aspell-fo aspell-fr aspell-ga aspell-gl-minimos aspell-gu aspell-he aspell-hi aspell-hr aspell-hsb aspell-hu aspell-hy aspell-id aspell-is aspell-it aspell-kk aspell-kn aspell-ku aspell-lt aspell-lv aspell-ml aspell-mr aspell-nl aspell-nr aspell-ns  aspell-pa aspell-pl aspell-pt aspell-pt-br aspell-ro aspell-ru aspell-sk aspell-sl aspell-ss aspell-st aspell-sv aspell-tl aspell-tn aspell-ts aspell-uk aspell-uz aspell-xh aspell-zu \
+    \
+# install Node.JS 10
+&&  curl -sSL https://deb.nodesource.com/setup_10.x | bash - \
+&&  apt-get install -y nodejs \
+    \
+&&  rm -rf \
+      /etc/nginx/nginx.conf \
+      /etc/nginx/sites-enabled/default \
+      /var/lib/apt/lists/*
 
 
 # Install qpdf
 # ------------
-WORKDIR /opt
-RUN wget https://s3.amazonaws.com/sharelatex-random-files/qpdf-6.0.0.tar.gz && tar xzf qpdf-6.0.0.tar.gz
-WORKDIR /opt/qpdf-6.0.0
-RUN ./configure && make && make install && ldconfig
-
-
-# Install Node
+RUN mkdir /opt/qpdf \
+    \
+&&  curl -sSL \
+      https://s3.amazonaws.com/sharelatex-random-files/qpdf-6.0.0.tar.gz \
+    | tar -xzC /opt/qpdf --strip-components=1 \
+    \
+&&  cd /opt/qpdf/ \
+&&    ./configure \
+&&    make \
+&&    make install \
+&&    ldconfig \
+    \
```

Comment:
Just personal taste, so ignore as you wish!: I find this backslash-only lines a bit noisy, indentation works just well for me with `&& \` at the end of the line

Reply:
The idea here was to split the different tasks into separate blocks: environment setup, downloading, compiling and cleanup.

When using the debian package, these steps can be dropped.